package.json

```diff
@@ -27,16 +26,17 @@
     "sinon": "^1.17.3"
   },
   "scripts": {
-    "lodash-custom": "lodash include=keys,defaults,remove exports=node -o ./lib/lodash.custom.js",
-    "build-debug": "browserify alephbet.js.coffee -d --standalone AlephBet -o dist/alephbet.js",
-    "build-min": "browserify alephbet.js.coffee --standalone AlephBet | uglifyjs -c > dist/alephbet.min.js",
-    "build": "npm run lodash-custom && npm run build-debug && npm run build-min",
-    "test-standalone": "browserify test/alephbet_test.coffee > test/bundle.js && echo '<script src=\"bundle.js\"></script>' > test/test.html",
-    "test": "browserify test/alephbet_test.coffee | tape-run",
-    "watch": "watchify alephbet.js.coffee -d --standalone AlephBet -o dist/alephbet.js -v"
+    "lodash-custom": "lodash -d include=keys,defaults,remove exports=node -o ./vendor/lodash.custom.js",
```

Comment:
Ah, ok, then I would use `-p` which creates only the minified version instead of `-d`, ok?